source/modules/MEI.usersymbols.xml

```diff
@@ -368,6 +368,7 @@
     <desc xml:lang="en">Contains a set of user-defined symbols.</desc>
     <classes>
       <memberOf key="att.common"/>
+      <memberOf key="att.linking"/>
```

Comment:
Why is this change here?

Reply:
We have added att.linking to allow for specification of `@target` and `@xlink:show` to load in externally-stored symbol table information, so we can share symbolTables across encodings that have the same use of glyphs.

Reply:
At this point, I think this explanation is sufficient. We should add a section in the documentation, as soon as e.g. SMUFL has added the corresponding symbols.